gslib/tests/test_copy_helper_funcs.py

```diff
@@ -111,14 +111,15 @@ def testGetPartitionInfo(self):
     self.assertEqual(50, component_size)
 
   def testFilterExistingComponentsNonVersioned(self):
-    """Tests upload with a variety of component states."""
+    """Tests upload with a variety of component states, including custom storage class."""
```

Comment:
Is this docstring change still necessary?

Reply:
No, I kept it to be more specific but I will delete it.